### DISCUSSION TITLE
r/city-china-shanghai-tangxiagonglu

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#54] [Announcements]
# r/city-china-shanghai-tangxiagonglu

游记｜塘下公路，隐藏在上海的绝美风景线

https://blog.ops-coffee.cn/r/city-china-shanghai-tangxiagonglu

**Comment by anghunk** [comment]
每期必看，羡慕博主的生活

**Comment by leetomlee123** [comment]
房车QaQ

**Comment by poezhang** [comment]
这大马路看的人神清气爽的

**Comment by beyond1314** [comment]
我本人就住在临港这边，隋唐河路基本都是去骑行，路上车子少。
至于塘下公路也是开车经过，不过夏天，秋天确实很美。
